### PR TITLE
Update styles of login and sign-up pages

### PR DESCRIPTION
**Pull request by morkune**
Updated styles in a simple way.

### CODE REVIEW

File: views/login.ejs [hunk 1]
```diff
@@ -1,11 +1,28 @@
 <%- include("./partials/header") %>
 
-<h1>Login</h1>
+<div class="container">
+    <div class="row">
+        <div class="col">
+            <h1 style="text-align: center; margin-top: 30px;">Login</h1>
```
Comment (julykaz) [inline views/login.ejs]: It's better not to use inline styles in HTML markup. I'd suggest creating a CSS class for styling.
Same about the next line.

File: views/login.ejs [hunk 2]
```diff
@@ -1,11 +1,28 @@
...
+            <div style="width: 30%; margin: 30px auto;">
+                <form action="/login" method="POST">
+                    <div class="form-group">
+                        <label for="exampleInputUsername1">Username</label>
+                        <input class="form-control" type="text" name="username" id="exampleInputUsername1">
```
Comment (julykaz) [inline views/login.ejs]: Why the ID is `exampleInputUsername1`? How it's an example?
You might not need the number at the end of the ID too.
Also - I'd suggest using kebab-case IDs.

File: views/register.ejs [hunk 3]
```diff
@@ -1,11 +1,27 @@
...
-
-<form action="/register" method="POST">
-<input type="text" name="username" placeholder="username">
-<input type="password" name="password" placeholder="password">
-<button>Sign up</button>
-</form>
+<div class="container">
+    <div class="row">
+    <div class="col">
+        <h1 style="text-align: center; margin-top: 30px;">Sign Up</h1>
```
Comment (julykaz) [inline views/register.ejs]: No inline styles. please.

File: views/register.ejs [hunk 4]
```diff
@@ -1,11 +1,27 @@
...
+        <div style="width: 30%; margin: 30px auto;">
+            <form action="/register" method="POST">
+                <div class="form-group">
+                    <label for="exampleInputUsername">Username</label>
+                    <input class='form-control' type="text" name="username" id="exampleInputUsername">
```
Comment (julykaz) [inline views/register.ejs]: ID could have a better name.

File: views/login.ejs (hunk 2 above)
Comment (morkune) [reply views/login.ejs]: It would be better to write login-input-username?

Comment (julykaz) [reply views/login.ejs]: It could be just `username` or `username-input`.

File: public/stylesheets/main.css [hunk 5]
```diff
@@ -12,4 +12,17 @@
...
+.form-heading {
+    text-align: center;
+    margin-top: 30px;
+}
+
+.form-label {
+    width: 30%;
+    margin: 30px auto;
+}
+
```
Comment (julykaz) [inline public/stylesheets/main.css]: A single blank line at the end of the file is enough.

Comment (morkune) [reply public/stylesheets/main.css]: Okay, I will leave it in the next commit.

File: public/stylesheets/main.css [hunk 6]
```diff
@@ -12,4 +12,14 @@
...
+.form-heading {
+    text-align: center;
+    margin-top: 30px;
+}
+
+.form {
+    width: 30%;
+    margin: 30px auto;
 }
```
Comment (julykaz) [inline public/stylesheets/main.css]: Having a single blank line at the end of the file is a common practice.

File: views/register.ejs [hunk 7]
```diff
@@ -1,11 +1,27 @@
...
-<h1>Sign up</h1>
-
-<form action="/register" method="POST">
-<input type="text" name="username" placeholder="username">
-<input type="password" name="password" placeholder="password">
-<button>Sign up</button>
-</form>
+<div class="container">
+    <div class="row">
+    <div class="col">
```
Comment (julykaz) [inline views/register.ejs]: Indentation is a bit off. Do not forget to format code before pushing it to the repo.

File: public/stylesheets/main.css (hunk 6 above)
Comment (morkune) [reply public/stylesheets/main.css]: Thank you! I already changed it, but I don't want to commit it now, just for this change.

**Pull request merged**